service_container/autowiring.rst

```diff
@@ -378,15 +409,21 @@ that alias:
             App\Util\Rot13Transformer: ~
             App\Util\UppercaseTransformer: ~
 
+            # the ``App\Util\UppercaseTransformer`` service will be
+            # injected when an ``App\Util\TransformerInterface``
+            # type-hint for a ``$shoutyTransformer`` argument is detected.
             # the ``App\Util\Rot13Transformer`` service will be injected when
-            # a ``App\Util\TransformerInterface`` type-hint is detected
+            # an ``App\Util\TransformerInterface`` type-hint is detected,
+            # but the argument name does not match that of a named alias
```

Comment:
A negative statement might not be clear enough. What about splitting the comment on top of its configuration line also?